Source/WebCore/rendering/style/RenderStyle.h

```diff
@@ -2447,6 +2448,13 @@ inline bool RenderStyle::isDisplayRegionType() const
         || display() == DisplayType::ListItem;
 }
 
+inline bool RenderStyle::isDisplayBlockLevel() const
```

Comment:
Maybe add a link to https://wicg.github.io/scroll-to-text-fragment/#has-block-level-display?